ssl-tool-win/openssl.cnf

```diff
@@ -90,6 +90,17 @@ keyUsage = critical, digitalSignature, keyEncipherment
 extendedKeyUsage = serverAuth, clientAuth
 subjectAltName = @alt_names
 
+[ server_cert ]

```

Comment:
This may break existing certificates generation process:
https://github.com/Alfresco/alfresco-ssl-generator/pull/18

Reply:
I've added it, but in script the extension chosen depends on role assigned through a parameter. If no role is assigned "both" is used and it defaults to "clientServer_cert" extension provided in link you've mentioned. Alfresco repository and Solr are using the clientServer_cert also the "new_simple" sample uses "both" role for all certs aside from the browser certificate.